autoemulate/compare.py

```diff
@@ -368,6 +378,74 @@ def load_model(self, path=None):
 
         return serialiser._load_model(path)
 
+    def print_settings(self):
+        if not self.is_set_up:
+            raise RuntimeError("Must run setup() before print_settings()")
+            return
+
+        models = "\n- " + "\n- ".join(
+            [
+                x[1].__class__.__name__
+                for pipeline in self.models
+                for x in pipeline.steps
+                if x[0] == "model"
+            ]
+        )
+        metrics = "\n- " + "\n- ".join([metric.__name__ for metric in self.metrics])
+
+        settings = pd.DataFrame(
+            [
+                str(self.X.shape),
+                str(self.y.shape),
+                str(self.train_idxs.shape),
+                str(self.test_idxs.shape),
```

Comment:
```
str(self.train_idxs.shape[0]),
str(self.test_idxs.shape[0]),
```

To get an int instead of tuple, see below.

Reply:
Addresed in https://github.com/alan-turing-institute/autoemulate/pull/195/commits/75d5ae61506dd76c7dd2ad3157c3cdc05c4e19e9